spartan/aztec-network/templates/boot-node.yaml

```diff
@@ -107,22 +110,31 @@ spec:
             - name: SEQ_MIN_TX_PER_BLOCK
               value: "{{ .Values.bootNode.sequencer.minTxsPerBlock }}"
             - name: P2P_TCP_ANNOUNCE_ADDR
-              value: "$(POD_DNS_NAME):{{ .Values.bootNode.service.p2pPort }}"
+              {{- if .Values.bootNode.externalTcpHost }}
+              value: "{{ .Values.bootNode.externalTcpHost }}:{{ .Values.bootNode.service.p2pTcpPort }}"
+              {{- else }}
+              value: "$(POD_DNS_NAME):{{ .Values.bootNode.service.p2pTcpPort }}"
+              {{- end }}
             - name: P2P_UDP_ANNOUNCE_ADDR
-              value: "$(POD_DNS_NAME):{{ .Values.bootNode.service.p2pPort }}"
+              {{- if .Values.bootNode.externalUdpHost }}
+              value: "{{ .Values.bootNode.externalUdpHost }}:{{ .Values.bootNode.service.p2pUdpPort }}"
+              {{- else }}
+              value: "$(POD_DNS_NAME):{{ .Values.bootNode.service.p2pUdpPort }}"
+              {{- end }}
             - name: P2P_TCP_LISTEN_ADDR
-              value: "0.0.0.0:{{ .Values.bootNode.service.p2pPort }}"
```

Comment:
Any reason to make these ports not always the same?

Reply:
Yes, when I was creating the load balancer, I had to create a separate one explicitly for UDP. Maybe it would've worked with two separate load balancers going over the same port, but I didn't try.

Reply:
all good with me, cleaner probably